doc/source/code_editors.rst

```diff
@@ -1,3 +1,35 @@
 ============
 Code Editors
 ============
+===================
+IDE Recommendations
+===================
+-NotepadPP
+	NotepadPP version 8.2 and the addon “reStructuredText_NPP”
+	Use the following link to GitHub for further instructions 
+	https://github.com/steenhulthin/reStructuredText_NPP
+
+-Vim 
+	Vim is a markdown editor which can be utilized for RST. For less experienced Users it would be better to use Visual Studio Code. VS Code is more user friendly in UI Design and overall experience
```

Comment:
vim is not a markdown editor - it is a highly extensible text editor for the textline interface. It also has a visual analogue: gvim (graphical vim). It is available mostly on every linux OS by default and is therefore a good candidate for remote editing

Reply:
please also add https://github.com/nvie/vim-rst-tables/ as a plugin